Tests/FluentMongoDriverTests/FluentMongoDriverTests.swift

```diff
@@ -252,8 +252,19 @@ final class FluentMongoDriverTests: XCTestCase {
             .flatMap { String(cString: $0) }
             ?? "localhost"
         try self.dbs.use(.mongo(connectionString: "mongodb://\(hostname):27017/vapor-database"), as: .mongo)
```

Comment:
Is this a (common) usecase? If people want to register multiple databases of the same type, in this case MongoDB, I'd prefer running those databases over the same connection if they're on the same server. In which case I feel that there should be a way to provide a MongoKitten `MongoDatabase`.